docs/configuration/index.md

```diff
@@ -1644,7 +1644,8 @@ The Druid SQL server is configured through the following properties on the Broke
 |`druid.sql.enable`|Whether to enable SQL at all, including background metadata fetching. If false, this overrides all other SQL-related properties and disables SQL metadata, serving, and planning completely.|true|
 |`druid.sql.avatica.enable`|Whether to enable JDBC querying at `/druid/v2/sql/avatica/`.|true|
 |`druid.sql.avatica.maxConnections`|Maximum number of open connections for the Avatica server. These are not HTTP connections, but are logical client connections that may span multiple HTTP connections.|25|
-|`druid.sql.avatica.maxRowsPerFrame`|Maximum number of rows to return in a single JDBC frame. Setting this property to -1 indicates that no row limit should be applied. Clients can optionally specify a row limit in their requests; if a client specifies a row limit, the lesser value of the client-provided limit and `maxRowsPerFrame` will be used.|5,000|
+|`druid.sql.avatica.maxRowsPerFrame`|Maximum value which can be supplied to the JDBC `Statement.setFetchSize` method. This setting determines the maximum sized JDBC `ResultSet` which Druid will populate in a single 'fetch'. Setting this property to -1 indicates that no row limit should be applied on the server-side, potentially returning the entire set of rows on the initial statement execution. If the JDBC client calls `Statement.setFetchSize` with a value other than -1, the lesser value of the client-provided limit and `maxRowsPerFrame` will be used. If `maxRowsPerFrame` is smaller than `minRowsPerFrame`, then the `ResultSet` size will be fixed. For handling queries which produce results with a large number of rows, consider increasing this value to reduce the number of fetches required to transfer the result set.|5,000|
+|`druid.sql.avatica.minRowsPerFrame`|Minimum value which can be supplied to the JDBC `Statement.setFetchSize` method. This property must be set to a value greater than 0. If a `Statement.setFetchSize` is specified with a smaller value, this row count will be used instead. If `maxRowsPerFrame` is set to a value lower than `minRowsPerFrame`, the minimum value of the two will be used. For handling queries which produce results with a large number of rows, consider increasing this value to reduce the number of fetches required to transfer the result set.|100|
```

Comment:
```suggestion
|`druid.sql.avatica.minRowsPerFrame`|Minimum acceptable value for the JDBC client `Statement.setFetchSize` method. The value for this property must greater than 0. If the JDBC client specifies a `Statement.setFetchSize` with a lesser value, Druid uses this row count instead. If `maxRowsPerFrame` is less than `minRowsPerFrame`, Druid uses the minimum value of the two. For handling queries which produce results with a large number of rows, you can increase this value to reduce the number of fetches required to completely transfer the result set.|100|
```

Reply:
Applied this suggestion with a couple of minor adjustments ("If the JDBC client specifies a `Statement.setFetchSize` with a lesser value, Druid uses this row count instead." -> "If the JDBC client calls `Statement.setFetchSize` with a lesser value, Druid uses `minRowsPerFrame` instead."), thanks